src/script/backport-create-issue

```diff
@@ -39,8 +39,11 @@ from redminelib.exceptions import ResourceAttrError
 redmine_endpoint = "https://tracker.ceph.com"
 project_name = "Ceph"
 release_id = 16
+custom_field_tag = 'cf_3'
+tag_backport_processed = 'backport_processed'
 delay_seconds = 5
 redmine_key_file="~/.redmine_key"
+redmine_key_env="REDMINE_API_KEY"
```

Comment:
```suggestion
redmine_key_env="$REDMINE_API_KEY"
```

Reply:
I don't think this is needed as this is passed to `os.getenv`.  `$` is appended when printing.